### PR TITLE
Update pk.srt

### PR DESCRIPTION
**Pull request by toinhoalam**
Two more scenes translated.

### CODE REVIEW

File: musi/pk.srt [hunk 1]
```diff
@@ -2595,207 +2595,271 @@ please.
...
 595
 01:11:42,842 --> 01:11:45,104
-open the door, i have an entry.
+o open e lupa. mi wile tawa.
+==open the door, i have an entry.
 
 596
 01:11:45,540 --> 01:11:49,224
+sewi Siwa. ona li pakala e ike
+li monsuta e ala.
```
Comment (stefichjo) [inline musi/pk.srt]: No neologisms, please: (i.e. no "monsuta").
→ li pilin akesi tan ala.
→ ala li pana e pilin akesi tawa ona.

File: musi/pk.srt [hunk 2]
```diff
@@ -2595,207 +2595,271 @@ please.
...
-didn't recognise me?
+tenpo ni la sina sona ala anu seme?
+mi jan Pike.
+==didn't recognise me?
 I am PK.
 
 599
 01:11:57,246 --> 01:12:03,762
-Security. anybody there?
+jan lawa o! jan li lon ala lon ni?
```
Comment (stefichjo) [inline musi/pk.srt]: ... anu seme?

File: musi/pk.srt [hunk 3]
```diff
@@ -2595,207 +2595,271 @@ please.
...
+jan nasa li lon ni. mi wile ala wile pana e musi lon poki pi ko jaki?
```
Comment (stefichjo) [inline musi/pk.srt]: → ... anu seme?

File: musi/pk.srt [hunk 4]
```diff
@@ -2595,207 +2595,271 @@ please.
...
 604
 01:12:13,437 --> 01:12:14,191
-spaceship?
+tomo tawa mun?
+==spaceship?
 
 605
 01:12:14,350 --> 01:12:18,190
-Forgotton? I told you so many times.
+sina weka ala weka? tenpo mute la mi toki e ijo ona.
```
Comment (stefichjo) [inline musi/pk.srt]: → ... anu seme?

File: musi/pk.srt [hunk 5]
```diff
@@ -2595,207 +2595,271 @@ please.
...
 01:12:18,477 --> 01:12:22,290
-I am not from this planet,
+mi tan ma ni ala. mi tan ma pi poka ala.
+==I am not from this planet,
 came from far away.
 
 607
 01:12:22,953 --> 01:12:26,564
-Just get me my remote and i'll go home.
+sina pana e ilo lawa tawa mi la mi tawa tomo mi.
```
Comment (stefichjo) [inline musi/pk.srt]: → **,** la

File: musi/pk.srt [hunk 6]
```diff
@@ -2595,207 +2595,271 @@ please.
...
 609
 01:12:30,374 --> 01:12:31,302
-i know it very well.
+mi sona pona e ni.
+==i know it very well.
 
 610
 01:12:31,747 --> 01:12:34,477
-Ganesh and kartik. They aren't little and can survive alone.
+sewi Kanesi en sewi Kateki. ona li lili ala li ken lon poka ala sina.
```
Comment (stefichjo) [inline musi/pk.srt]: → Kane
→ Kasi/Kate

File: musi/pk.srt [hunk 7]
```diff
@@ -2595,207 +2595,271 @@ please.
...
+==Ganesh and kartik. They aren't little and can survive alone.
 
 611
 01:12:34,477 --> 01:12:38,955
-you get me my remote and then
+sina pana e ilo lawa mi la sina ken
```
Comment (stefichjo) [inline musi/pk.srt]: → **,** la

File: musi/pk.srt [hunk 8]
```diff
@@ -2595,207 +2595,271 @@ please.
...
-..then suddenly i see light,
+ni la suno li kama
+tan ijo lili ni lon anpa.
+==..then suddenly i see light,
 from this small object on the ground.
 
 620
 01:14:11,493 --> 01:14:19,236
-I closed my eyes thought about God,
+mi pini e oko mi li pilin e sewi.
```
Comment (stefichjo) [inline musi/pk.srt]: oko → lukin

File: musi/pk.srt [hunk 9]
```diff
@@ -2595,207 +2595,271 @@ please.
...
-God confirmed its the broken piece of
+sewi li toki e ni: ijo ni li wan tan
+ilo kalama pi lawa Siwa. o kama jo e ona.
+==God confirmed its the broken piece of
 lord shiva's pellet drum. embrace it.
 
 623
 01:14:36,017 --> 01:14:48,306
-take it and put it in a temple. its auspicious
+li pana e ona lon tomo sewi. lukin ona li kama
```
Comment (stefichjo) [inline musi/pk.srt]: li pana → o pana

File: musi/pk.srt [hunk 10]
```diff
@@ -2595,207 +2595,271 @@ please.
...
+e pona tawa jan li weka e ike tan jan.
+==take it and put it in a temple. its auspicious
 sight will take people's trouble away.
 
 624
 01:14:49,985 --> 01:14:57,459
-would you like to see it?
+sina wile ala wile lukin e ona?
```
Comment (stefichjo) [inline musi/pk.srt]: ... anu seme?

File: musi/pk.srt [hunk 11]
```diff
@@ -2595,207 +2595,271 @@ please.
...
 630
 01:15:31,489 --> 01:15:33,711
-not there, here
+o tawa ni ala. o kama.
+==not there, here
 
 631
 01:15:33,949 --> 01:15:36,383
-Say what you want to say here.
+o kama li toki e ijo pi wile sina.
```
Comment (stefichjo) [inline musi/pk.srt:2809]: li → o

File: musi/pk.srt [hunk 12]
```diff
@@ -2595,207 +2595,271 @@ please.
...
+ijo ni li seli mute. taso ona
+li lon lete mute.
+==this object was so hot even in
 a very cold weather.
 
 622
 01:14:28,080 --> 01:14:35,795
-God confirmed its the broken piece of
+sewi li toki e ni: ijo ni li wan tan
+ilo kalama pi lawa Siwa. o kama jo e ona.
```
Comment (stefichjo) [inline musi/pk.srt]: "lawa" is the head, but not the leader. (This happens repeatedly in this scene.)
lawa → jan lawa

File: musi/pk.srt [hunk 13]
```diff
@@ -2595,207 +2595,271 @@ please.
...
+mi lukin e ona lon tomo pi ko jaki.
+mi wile kama jo e ijo mi la ona li tawa wawa.
+==He's right here.
 found him in the toilet.
 when i asked for my property, he ran.
 
 635
 01:15:54,584 --> 01:16:03,182
-I thought he escaped but he brought me here
+mi la ona li weka. taso ona li pana e mi lon ni
```
Comment (stefichjo) [inline musi/pk.srt]: "mi la" is *"lon mi la", but "tawa mi la" is explained in pu, whereas "mi la" isn't.
→ tawa mi la

File: musi/pk.srt [hunk 14]
```diff
@@ -2595,207 +2595,271 @@ please.
...
+tawa ijo mi.
+o sewi e lawa!
+==I thought he escaped but he brought me here
 to my property.
 praise the lord!
 
 636
 01:16:03,791 --> 01:16:09,861
-Its not shiva's. its mine.
+sewi Siwa li jo ala e ni. mi kin li jo.
```
Comment (stefichjo) [inline musi/pk.srt]: kin → a

File: musi/pk.srt [hunk 15]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
 644
 01:18:15,370 --> 01:18:18,780
-you think my story is fake?
+tawa sina la toki mi li lon ala anu seme?
+==you think my story is fake?
 
 645
 01:18:21,323 --> 01:18:28,362
-so go on telecast nikku dog story
+tan ni la o tawa li pana e toki pi soweli Niku
```
Comment (stefichjo) [inline musi/pk.srt:2895]: li → o

File: musi/pk.srt [hunk 16]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+lon ilo pana. ijo pona li kama ala lon nasin sina.
+==so go on telecast nikku dog story
 on tv. you aren't lucky enough.
 
 646
 01:18:36,164 --> 01:18:39,585
-pandey ji. I want to meet him once
+jan Panisi o, mi wile toki e ijo sin tawa ona.
```
Comment (stefichjo) [inline musi/pk.srt]: → Pante Si

File: musi/pk.srt [hunk 17]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+==pandey ji. I want to meet him once
 more.
 
 647
 01:18:40,132 --> 01:18:49,030
-Instead give me 500 rupee more
+sina pana e mani namako la
```
Comment (stefichjo) [inline musi/pk.srt]: namako → sin
→ **,** la

File: musi/pk.srt [hunk 18]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+mi pana kin e ona tawa sina.
```
Comment (stefichjo) [inline musi/pk.srt]: kin → a

File: musi/pk.srt [hunk 19]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
-i don't remember telling you about
+tawa mi la mi toki ala e toki pi soweli
+pi lukin moli tawa sina.
+==i don't remember telling you about
 suicidal dog.
 
 649
 01:18:52,354 --> 01:18:53,920
-because you didn't.
+sina toki ala kin.
```
Comment (stefichjo) [inline musi/pk.srt]: kin → a

File: musi/pk.srt [hunk 20]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
 650
 01:18:54,259 --> 01:18:56,209
-how did you know?
+sina sona kepeken nasin seme?
+==how did you know?
 
 651
 01:18:57,310 --> 01:19:02,299
-I read your mind when you held my hand.
+sina jo e luka mi la mi lukin e insa pi lawa sina.
```
Comment (stefichjo) [inline musi/pk.srt]: → **,** la

File: musi/pk.srt [hunk 21]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
-you thought my story is bogus,
+tawa sina la toki mi li lon ala. tan ni la
+sina wile toki e ijo pi soweli Niku.
+==you thought my story is bogus,
 you have to manage with nikku dog now.
 
 653
 01:19:06,481 --> 01:19:09,418
-don't lie. i must have told you.
+o toki e lon! mi toki kin e ona tawa sina.
```
Comment (stefichjo) [inline musi/pk.srt]: kin → a

File: musi/pk.srt [hunk 22]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+mije sin o pana e pona. meli mi li ike lon tomo pi pona sijelo.
+ona li wile e mani pi mute suli tawa pona ona.
+mi wile e mani lili taso.
+==Help me son. My wife is sick in the hospital.
 they need 10000 rupees.
 I am short with 500 rupees.
 
 658
 01:19:28,498 --> 01:19:33,184
+sina pana e mani lili ni la...
```
Comment (stefichjo) [inline musi/pk.srt]: → **,** la

File: musi/pk.srt [hunk 23]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
 661
 01:19:41,672 --> 01:19:43,981
-Thank you son.
+mije sin o pona mute.
+==Thank you son.
 
 662
 01:19:44,133 --> 01:19:48,515
-Uncle! keep 100 rupee more
+jan suli o kama jo e mani lili namako
```
Comment (stefichjo) [inline musi/pk.srt]: namako → sin

File: musi/pk.srt [hunk 24]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+tawa pana namako.
```
Comment (stefichjo) [inline musi/pk.srt]: namako → sin

File: musi/pk.srt [hunk 25]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+==Uncle! keep 100 rupee more
 for the tip.
 
 663
 01:19:49,642 --> 01:19:53,167
-long live son.
+ali pona tawa sina!
```
Comment (stefichjo) [inline musi/pk.srt]: ali → ale

File: musi/pk.srt [hunk 26]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+==long live son.
 
 664
 01:19:53,498 --> 01:20:03,898
-its proved now that you were lying.
+sina toki e lon ala kin. ona li nasa e sina.
```
Comment (stefichjo) [inline musi/pk.srt]: kin → a

File: musi/pk.srt [hunk 27]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+tenpo suno ni la meli ona li sike suno sin.
+tenpo pini ala la ona li lon tomo pi mun luka.
+==Today his wife's 75th birthday.
 She's never been to 5 star hotel.
 
 667
 01:20:13,314 --> 01:20:19,026
-gradually he was collecting money from
+tenpo suli la ona li awen e mani pi kama jo ona.
+meli ona li wile e suwi lete namako.
```
Comment (stefichjo) [inline musi/pk.srt]: namako → sin/-

File: musi/pk.srt [hunk 28]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
+==gradually he was collecting money from
 his pension. wife ordred an extra ice cream.
 
 668
 01:20:19,026 --> 01:20:22,783
-he went out of money. he gave bathroom as
+taso ona li jo ala kin e mani. toki ona la
```
Comment (stefichjo) [inline musi/pk.srt]: kin → a

File: musi/pk.srt [hunk 29]
```diff
@@ -2822,135 +2886,182 @@ meet a psychiatrist. call me.
...
-in our ball nobody lies.
+sike mi la jan li toki e lon taso.
+pilin sina li ante ala e ni.
+==in our ball nobody lies.
 believe it or not.
 
 671
 01:21:36,360 --> 01:21:41,030
-Its your birthday after all.
+ni li sike suno sina kin.
```
Comment (stefichjo) [inline musi/pk.srt]: kin → a

**Review by stefichjo — CHANGES_REQUESTED** [review]

File: musi/pk.srt (hunk 1 above)
Comment (toinhoalam) [reply musi/pk.srt]: pilin akesi is a terrible expression that doesn't make sense. pilin ike is better.

**Comment by toinhoalam** [comment]
Sorry for this. I reviewed and translated two more scenes this afternoon, but I must've forgotten to save them. Please do scrape through my commits looking for the needed adjustments I missed.

File: musi/pk.srt (hunk 1 above)
Comment (stefichjo) [reply musi/pk.srt]: For me "pilin akesi" fits. "pilin ..." in general is a very strange construction used in Toki Pona. (I will have to think about it one more time.) But yes, "pilin ike" is fine.

**Review by stefichjo — APPROVED** [review]

**Pull request merged**